docs/src/pages/demos/tables/SimpleTable.js

```diff
@@ -19,10 +19,8 @@ const styles = theme => ({
   },
 });
 
-let id = 0;
 function createData(name, calories, fat, carbs, protein) {
-  id += 1;
```

Comment:
That's a good change. The previous approach resulted in re-mounting of the `TableRow` on every render call of `SimpleTable`.